src/values.rs

```diff
@@ -67,11 +76,50 @@ impl Type {
         use self::Type::*;
         match *self {
             Duration | TimeStamp | Temperature | ExtNumeric | Color => false,
-            Unit | Bool | String | Json | Binary => true,
+            Unit | String | Json | Binary | OnOff | OpenClosed | ExtBool  => true,
         }
     }
 }
 
+/// An on/off state. Internal representation may be either On or Off.
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum OnOff {
+    On,
+    Off,
+}
+
+impl OnOff {
```

Comment:
Hey @Yoric I need your help/advise here :)

I'm trying to better understand ```PartialOrd``` for different types, and I was thinking that boolean-like enums should implement the same traits as real bool - ```Eq``` in addition to ```PartialEq``` and ```PartialOrd```, also I found that having ```as_bool``` method can be handy (maybe there should be ```BoolValue {fn as_bool}``` trait), but still not everything clear in my head - so please let me know if I over-complicating things or introducing bloat.

Thanks!